helm/dex-app/templates/dex-k8s-authenticator/ingress.yaml

```diff
@@ -11,8 +11,9 @@ metadata:
     {{- include "dexk8sauth.labels.common" . | nindent 4 }}
   annotations:
     {{- if .Values.ingress.tls.letsencrypt }}
-    kubernetes.io/tls-acme: "true"
```

Comment:
this is old annotations no longer necessary if the `cluster-issuer` annotations is used